### DISCUSSION TITLE
Default Suricata rulset(s) in SO

### DISCUSSION THREAD

**Discussion by ll3N1GmAll** [Security-Onion-Solutions/securityonion#4720] [Unsupported Versions] [answered]
What are the default rulesets for Suricata in SO? I am running SO internally without active blocking (I have Suricata at the perimeter already as a part of my router hardware). However, SO is seeing way more. I would like to add the SO Suricata rules to my router instance of Suricata. What are they all so I can add them?

**Comment by weslambert** [comment, marked as answer]
Security Onion defaults to using ET Open for the NIDS ruleset. You might be seeing more alerts than from your router because of what traffic is being seen and how it relates to HOME_NET and EXTERNAL_NET.

**Reply by ll3N1GmAll** [reply]
Thank you for your reply.
